Chainlink

```diff
@@ -0,0 +1,236 @@
+### **Chainlink Blockchain Governance**
+
+**Overview**
+
+ChainLink is blockchain middleware that allows smart contracts to securely connect to various APIs, resources, and bank accounts. This occurs through the ChainLink Network. The network is a decentralized system of nodes, better known as oracles, which sell usage of APIs and other resources to a smart contract. The selling of data to the contracts is done through the LINK token, an ERC20 token on the Ethereum Blockchain. The ChainLink team is made up of Sergey Nazarov, CEO, and Steve Ellis, CTO, with 6 advisors.
+
+Chainlink is [open source](https://github.com/smartcontractkit/chainlink/blob/master/LICENSE), and their upcoming decentralized network of nodes was created with specific token economics to make sure the node operators are incentivized to provide the correct answers, and get rewarded for being honest. Chainlink is designed to benefit all participants in its ecosystem:
+
+The Node Operators will be able to receive LINK tokens as form of payment for providing a 24/7 reliable source for the API feeds they choose to offer, and also the option to “stake” their LINK tokens for [collatera](https://github.com/smartcontractkit/chainlink/wiki/Protocol-Information#penalty-deposits)l, which will be requested in some service level agreements.
+
+The Smart Contracts Developers will have end-to-end reliable contracts by confirming the data feed was pulled from their desired source(s). The API Providers will benefit from a new lucrative way to sell their data.
+
+Chainlink nodes are not the conventional “stake and forget” nodes that we are used to from various Proof of Stake (“PoS”) coins, which require little to no participation once you have staked them. Chainlink nodes need to offer information which is provided via an API (Application Programming Interface).
```

Comment:
This text appears in [Altcoin Magazine](https://medium.com/altcoin-magazine/earning-your-way-in-the-trustless-future-chainlink-ecosystem-101-2922d56c9bfa), so should be a quote block with a link as reference.